packages/frontend/component/src/ui/avatar/avatar.stories.tsx

```diff
@@ -58,3 +58,16 @@ WithRemove.args = {
     console.log('on remove', e);
   },
 };
+
+export const CustomizeBorderRadius: StoryFn<AvatarProps> = args => (
+  <div style={{ display: 'flex', gap: 4 }}>
+    <Avatar {...args} size={60} colorfulFallback name="AFFiNE" rounded={10} />
+    <Avatar
+      {...args}
+      size={60}
+      url="https://app.affine.pro/api/workspaces/qf73AF6vzWphbTJdN7KiX/blobs/v7aTUh1LAwe_qyBxjyGjgEGeVGp2l1Ra0GAmHbM4HkI=.webp"
```

Comment:
seems not a stable resource url

Reply:
Replaced with a tiny base64(about `4kb`)